package.json

```diff
@@ -25,7 +25,7 @@
     "bundle": "rollup -i src/index.js -o dist/hyperapp.js -m -f umd -n hyperapp",
     "minify": "uglifyjs dist/hyperapp.js -o dist/hyperapp.js -mc pure_funcs=Object.defineProperty --source-map includeSources,url=hyperapp.js.map",
     "prepare": "npm run build",
-    "format": "prettier --semi false --write \"src/**/*.js\" \"{,test/ts/}*.{ts,tsx}\"",
+    "format": "prettier --semi false --write {src,test}/**/*.js {,test/ts/}*.{ts,tsx}",
```

Comment:
@okwolf Didn't you made the previous commit yourself to support windows or something?

Reply:
The previous change was from single quotes to double quotes, but it doesn't look like quotes are even needed at all.

Reply:
@okwolf Sure, not needed in unix.

Reply:
@JorgeBucaran verified still working on Windows too! 🎉